Source/CesiumRuntime/Private/CesiumLifetime.cpp

```diff
@@ -2,6 +2,11 @@
 
 #include "CesiumLifetime.h"
 #include "CesiumRuntime.h"
+#if WITH_EDITOR
+#include "Editor.h"
+#include "Editor/EditorEngine.h"
+#endif
+#include "Engine/Selection.h"
```

Comment:
```suggestion
#include "Engine/Selection.h"
#endif
```

I know that `Selection.h` isn't technically an Editor class, but maybe it would be nice to put it in this `if` anyway? We don't use the class outside an editor context anyway